pkg/aaparser/aaparser.go

```diff
@@ -22,6 +22,13 @@ func GetVersion() (int, error) {
 	return parseVersion(output)
 }
 
+// IsAvailable checks if the apparmor_parser binary is available.
+// This should be called before any other function from this package is called.
```

Comment:
I'm not sure this comment line still makes sense, as `InstallDefault` is checking for the consumer.

It also feels like a `sync.Once` might make sense here, and in `GetVersion` as well.

Reply:
I don't think `sync.Once` is appropriate here. Package containing the binary may be uninstalled after the first check was performed/or be installed after the first check happened.